inscriptions/ISic000694.xml

```diff
@@ -191,14 +196,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g>
+                    <lb n="2"/><name>Asinia</name> <name>Char
+                    <lb n="3" break="no"/>mosyne</name> <g ref="#interpunct">·</g> <expan><abbr>v</abbr><ex>ixit</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>a</abbr><ex>nnis</ex></expan> <g ref="#interpunct">·</g>
+                    <lb n="4"/><num value="65">LXV</num> <g ref="#interpunct">·</g> <expan><abbr>mens</abbr><ex>ibus</ex></expan> <g ref="#interpunct">·</g> <g ref="#interpunct">·</g> <hi rend="supraline"><num value="7">VII</num></hi> <g ref="#interpunct">·</g>
+                    <lb n="5"/><name>Mallianus</name>
```

Comment:
I think it is highly unlikely that this is anything other than M(arcus) Allianus

Reply:
Not a problem, was just following what was in the existing ISic Comm.